docs/usage.rst

```diff
@@ -79,8 +79,6 @@ For example::
 Your service should now be able to authenticate with Kerberos
 
 
-.. _pagination-label:
```

Comment:
This breaks the build.

Reply:
weird, I didn't change this in the last commit and its set up the same as usage.